README/ReleaseNotes/v626/index.md

```diff
@@ -81,6 +102,36 @@ More details at [PR #8737](https://github.com/root-project/root/pull/8737).
 - As an aid to `RDataSource` implementations with which collection sizes can be retrieved more efficiently than the full collection, `#var` can now be used as a short-hand notation for column name `R_rdf_sizeof_var`.
 - Helpers have been added to export data from `RDataFrame` to RooFit datasets. See the "RooFit Libraries" section below for more details.
 
+### Distributed RDataFrame
+The distributed RDataFrame module has been improved. Now it supports sending RDataFrame tasks to a [Dask](https://dask.org/) scheduler, for example:
+
+```python
+import ROOT
+from dask.distributed import Client
+RDataFrame = ROOT.RDF.Experimental.Distributed.Dask.RDataFrame
+
+# In a Python script the Dask client needs to be initalized in a context
+# Jupyter notebooks / Python session don't need this
+if __name__ == "__main__":
+    client = Client("SCHEDULER_ADDRESS")
+    df = RDataFrame("mytree","myfile.root", daskclient=client)
+    # Proceed as usual
+    df.Define("x","someoperation").Histo1D("x")
+```
+
+Through Dask, RDataFrame can be also scaled to a cluster of machines managed through a batch system like HTCondor or Slurm. Other notable additions and improvements include:
+
+- Set minimum Python version to use this tool to 3.7. This allows using more modern Python functionality in distributed RDataFrame code and is in line with the Python support provided by Spark and Dask.
+- Refactor triggering of the computation graph in the distributed tasks, so that it now runs with the Python GIL released. This allows interoperability with frameworks like Dask that run different Python threads along the main processing one.
+- Add support for the `DefinePerSample` operation.
+- Fixed a bug that disregarded user provided `npartitions` parameter in distributed Spark execution.
+- Refactor internal mechanism to store information about data sources and create ranges for the distributed tasks accordingly. This will allow in the future to easily extend the supported data sources in distributed RDataFrame.
+- Improve support for friend trees in distributed executions with TTree as data source. This fixes a long-standing issue with distributed RDataFrame [#7584](https://github.com/root-project/root/issues/7584).
+- Changed naming scheme of the files written by a distributed `Snapshot` operation. Now, each task will be assigned with a sequential id that will be appended to the name of the file created during the execution. For example, calling `Snapshot("mytree","myfile.root")` on a distributed RDataFrame with 3 partitions will create three files named like so: `myfile_0.root, myfile_1.root, myfile_2.root`.
+- Distributed execution with TTree data source now uses `TEntryList` in order to select the range of entries that each task will read from the tree. This highly reduces the waiting time spent in retrieving the correct entries for processing, as it was previously done using the `Range` operation.
```

Comment:
the highlight for users is "The distributed task overhead time has been reduced", you should probably mention that first, then optionally give some details about how it was done if you want.

And _great_ work overall!